openerp/addons/base/base_demo.xml

```diff
@@ -6,11 +6,14 @@
             <field name="name">Demo User</field>
             <field name="company_id" ref="main_company"/>
             <field name="customer" eval="False"/>
+            <field name="phone">+1 555-555-5555</field>
             <field name="email">demo@yourcompany.example.com</field>
-            <field name="street">Avenue des Dessus-de-Lives, 2</field>
-            <field name="city">Namur (Loyers)</field>
-            <field name="zip">5101</field>
-            <field name="country_id" ref="be"/>
+            <field name="street">YourCompany</field>
```

Comment:
I didn't know that the demo user already had an adress; you can keep the former one.